bucket/qimgv-video.json

```diff
@@ -0,0 +1,39 @@
+{
+    "version": "0.9.1",
+    "description": "Qt5 image viewer",
+    "homepage": "https://github.com/easymodo/qimgv",
+    "license": "GPL-3.0-only",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/easymodo/qimgv/releases/download/v0.9.1/qimgv-x64_0.9.1-video.zip",
+            "hash": "26c679a572159b5e405b213c6ea433a8a284fa9d7b79ae90bb3763bd04efd59d"
+        },
+        "32bit": {
+            "url": "https://github.com/easymodo/qimgv/releases/download/v0.9.1/qimgv_0.9.1-video.zip",
+            "hash": "13388ddf588e0feff712583daa0487497d827d37714b94d27c6a3d45b871ede0"
+        }
+    },
+    "extract_dir": "qimgv",
+    "bin": "qimgv.exe",
+    "shortcuts": [
+        [
+            "qimgv.exe",
+            "qimgv"
+        ]
+    ],
+    "persist": [
+        "conf",
```

Comment:
I would persist `cache` also

Reply:
That's fine with me, I will implement the changes now, but the non-video manifest does not do this.